.github/ISSUE_TEMPLATE/3_bug_report.yml

```diff
@@ -93,6 +93,7 @@ body:
     label: Kubernetes Version
     description: What version of Kubernetes that are you running?
     options:
+    - "1.30"
     - "1.29"
     - "1.28"
     - "1.27"
```

Comment:
Let's remove 1.26 and use `<1.27`

Reply:
If we set 1.28 as minimum, we have to drop 1.27 too